packages/ketcher-core/src/application/editor/tools/Bond.ts

```diff
@@ -465,21 +466,32 @@ class PolymerBond implements BaseTool {
 
     const atomRenderer = event.target.__data__ as AtomRenderer;
     const monomer = this.bondRenderer?.polymerBond.firstMonomer;
+
+    if (!this.isHydrogenBond && !monomer.chosenFirstAttachmentPointForBond) {
+      this.editor.events.error.dispatch(
+        'CHEM monomer to Atom supports only attachment points for bond creation',
```

Comment:
if new rule is applied not just for CHEM monomers, then please update this message

Reply:
Updated